preconditions/README.md

```diff
@@ -1,9 +1,11 @@
 # IOTICS PRECONDITIONS
 
-In order to guarantee the full streaming functionality of IOTICS, we would need:
-- **HTTP2** to be enabled with your environment;
-- web traffic to be allowed on ports **10001** (gRPC) and **11000** (gRPC-Web).
-- We would also need to ensure the domains `.iotics.space` and `did.prd.iotics.com` will not be blocked
+In order to guarantee the functionality of IOTICS, we would need to ensure the following services, ports and protocols are supported. This will need to be verified in corporate environments where firewalls and Internet proxies may affect successful operations:
+
+- REST | HTTP 1.1 | https://ganymede.iotics.space:443/qapi
+- STOMP over Websocket | HTTP 1.1 | wss://ganymede.iotics.space/ws
+- gRPC | HTTP 2 | https://ganymede.iotics.space:10001
+- gRPC Web | HTTP 2 / HTTP 1.1 | https://ganymede.iotics.space:11000
```

Comment:
Does gRPC-web needs to be tested over HTTP1 or HTTP2, or both ?

Reply:
well if you have tested that grpc doesn't work over http2 because http2 is blocked, the only thing that is left to test is grpcweb over http1.1
I was asking how you can enforce the grpc client to offer 1.1

Reply:
Via `curl --http1.1`